config/default.json

```diff
@@ -42,7 +42,14 @@
     "maxPayloadSizeMB": 50,
     "truncateSize": 15000,
     "truncateAppend": "\n[truncated ...]",
-    "authenticationTypes": ["basic", "local", "token"]
+    "authenticationTypes": ["basic", "local", "token", "openid"],
+    "keycloak": {
```

Comment:
The openid auth type isn't specifically restricted to keycloak. I wonder if we should rename this section to `oauth` or something similar. Then we should also not make so much mention of keycloak in the rest of the code. I don't see anything that ties us to keycloak right now.

Reply:
cool done :)